IsraelHiking.Web/src/scss/common.scss

```diff
@@ -97,20 +97,27 @@ body {
         min-width: unset !important;
         padding: 0 !important;
         display: block;
-        width: v.$app-button-size;
-        height: v.$app-button-size;
+        width: v.$app-button-width;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    .active:not(.active:first-child , .search .active ){
+        margin-top:7px;
+    }
 }
-
+.side-control.search .pull-left-important
```

Comment:
As far as I can see `pull-left-important` is only relevant to a single div, can this div receive a padding-top or margin top for the entire div instead of adding to every button?
If not, then I think it would be better to add a class to all the relevant buttons in that div.